src/components/home/HomeAboutMonarch.md

```diff
@@ -0,0 +1,107 @@
+<div class="home-page-section about-monarch-section">
+    <div class="about-block">
+       <h3> Connecting phenotypes to genotypes across species </h3>
+       <!-- <div class="data-section">
+          <div class="row">
+            <div class="col-sm">
+              <router-link
+                to="/analytics"
+                class="btn btn-link"
+                role="button">
+                <img
+                  class="icon"
+                  src="@/assets/img/monarch-ui-icon_ANATOMY.png"
+                  alt="disease">
+              </router-link>
+              <span class="network-name">
+                  <p class="amount">218,313</p>
+                  <p class="text">Disease-Phenotype Associations</p>
+              </span>
+            </div>
+            <div class="col-sm">
+              <router-link
+                to="/analytics"
+                class="btn btn-link"
+                role="button">
+                <img
+                  class="icon"
+                  src="@/assets/img/monarch-ui-icon_GENE.png"
+                  alt="disease">
+              </router-link>
+              <span class="network-name">
+              <p class="amount">793,526</p>
+              <p class="text">Gene-Phenotype Associations</p>
+            </div>
+            <div class="col-sm">
+              <router-link
+                to="/analytics"
+                class="btn btn-link"
+                role="button">
+                <img
+                  class="icon"
+                  src="@/assets/img/monarch-ui-icon_MODEL.png"
+                  alt="disease">
+              </router-link>
+              <span class="network-name">
+              <p class="amount">20,870</p>
+              <p class="text">Model Associations</p>
+            </div>
+          </div>
+        </div> -->
+        <div class="description">
+            Semantic-based integrative data analytics.<br><br>
+            <router-link to="/about/monarch">
+                <b-button squared class="about-btn">Learn More<i class="fa fa-caret-right"></i>
+                </b-button>
+            </router-link>
+        </div>
+    </div>
+</div>
+
+
+<style lang="scss">
+@import "~@/style/variables";
+@import "~@/style/home-page";
+
+div.about-monarch-section {
+  padding: 15px;
+  background-image: url("../../assets/img/network.jpg");
```

Comment:
@iimpulse I still don't love this image. It seems stretched out and just does not fit here. I'm sure we will get more feedback about this soon. 😉